### ISSUE TITLE
Category Name&Manufacturer  Drop down Broken After Upgrade to 6.1

### ISSUE THREAD

**Issue by spearzane** [grokability/snipe-it#13183]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

After upgrading from  v6.0.14 to v6.1.0 drop downs for both Manufacturer and Category Name do not load any of the information and just has a spinning dialog box.

### Reproduction steps

1. Upgrade to v6.1.0 and functionality breaks
2.Down Grade back to v6.0.14 and functionality is restored
3.
...

### Expected behavior

Drop down should return Manufactures and Category Names as they do currently before upgrading or after downgrading back to previous version. 

### Snipe-IT Version

6.1

### Operating System

Centos 7

### Web Server

Php

### PHP Version

8.0.28

**Comment by snipe** [comment]
Anything in your logs? Can you reproduce on the demo? Do you have full multiple company support enabled?

**Comment by spearzane** [comment]
We do not currently have full multiple company support enabled. I logged into the demo instance and it doesn't seem to be having the same issue as what we are. I also tried upgrading to the latest version that was just released 20 hours ago and that did not fix it either. When I search the  laravel.log file I don't see anything that is screaming out to me. It is just this error 
production.ERROR: file_put_contents(/var/www/mediatracker/public/vendor/livewire/livewire.js): Failed to open stream: Permission denied  But after changing the livewire perms back to nginx this error doesn't happen after rerunning the php upgrade.php command.
